go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -154,7 +154,7 @@ func DemystifyPending(status v1.PodStatus) (pluginsCore.PhaseInfo, error) {
 								// So be default if the container is not waiting with the PodInitializing/ContainerCreating
 								// reasons, then we will assume a failure reason, and fail instantly
 								t := c.LastTransitionTime.Time
-								return pluginsCore.PhaseInfoRetryableFailure(c.Reason, c.Message, &pluginsCore.TaskInfo{
+								return pluginsCore.PhaseInfoSystemRetryableFailure(c.Reason, c.Message, &pluginsCore.TaskInfo{
```

Comment:
Should we convert these into system errors:
https://github.com/lyft/flyteplugins/blob/779aab5d88b0add69358ab8269dee17fc87e1218/go/tasks/plugins/hive/execution_state.go#L108

https://github.com/lyft/flyteplugins/blob/c7dd2ebbc8982984422a91394d6490a76d2c2e7f/go/tasks/plugins/array/core/state.go#L215

Reply:
Done